### ISSUE TITLE
EntityTrigger does not get triggered in version of Microsoft.Azure.Functions.Worker.Extensions.DurableTask 1.1.3

### ISSUE THREAD

**Issue by netsharpdev** [microsoft/durabletask-dotnet#315]
There is an issue in version 1.1.3 that prevents triggering **EntityTrigger**. I have been calling SignalEntityAsync and CallEntityAsync and none of them ever reached the entity. When I downgraded the package to 1.1.1 it started working.

Here is the repository for reference and testing if you wish.
https://github.com/netsharpdev/g9-durable/

**Comment by th3ragex** [comment]
Could this also cause my locks to never be entered anymore? Worked with 1.1.2 .

```c#
var entityLock = new EntityInstanceId("SomeLock", somekey);
await using (await context.Entities.LockEntitiesAsync(entityLock))
{
...
}
```

**Comment by davidmrdavid** [comment]
@netsharpdev, @th3ragex: do y'all have **minimal** repros you could share?
@netsharpdev: I looked at your repro but it seemed a bit complex, so I'm curious if there's a more minimal version of it that demonstrates the problem. I'm not able to reproduce this with a "hello world"-level entity app

**Comment by th3ragex** [comment]
@davidmrdavid Sure, minimal repo with LockEntitiesAsync demo: https://github.com/th3ragex/durabletask-dotnet-bug-315 (Currently with 1.1.2 which works, after update to 1.1.3 and a proper rebuild it's stuck)

**Comment by bachuv** [comment]
@davidmrdavid, I tried the repro and it also worked for me after upgrading the .NET isolated extension version to 1.1.3.

**Comment by davidmrdavid** [comment]
Thanks for confirming. @th3ragex: When using 1.1.3 **and** a brand new taskhub name, does it still fail for you? Since it works for @bachuv and myself, I wonder if the reason for your lock's "stuckness" is that there's some actual deadlock in your storage account, not sure. A brand new task hub should help validate that theory.

**Comment by th3ragex** [comment]
@davidmrdavid thank for taking your time and testing

>When using 1.1.3 and a brand new taskhub name, does it still fail for you?
yes, I have also tested with azurite local storage and a gen2 azure storage with new task hub names. I am testing locally with the VS debugger attached, didn't try how it behaves as deployed azure function. I have also updated my machine to the latest azure function host and VS version.

After playing with my sample code a bit I discovered that building the function after upgrading to the new nuget version occasionally didn't use the new assembly version. I have changed my code  to return the loaded assembly version to make it clear.

Sorry to bother you, could you please update and try my repo again and double check that your test run actually used 1.1.3?

**Comment by davidmrdavid** [comment]
We just made a hotfix release: https://github.com/microsoft/durabletask-dotnet/releases/tag/v1.2.4. Is this issue persisting for you all with this release?

@nytian

**Comment by epDugas** [comment]
> We just made a hotfix release: https://github.com/microsoft/durabletask-dotnet/releases/tag/v1.2.4. Is this issue persisting for you all with this release?
> 
> cc/ @nytian

@davidmrdavid 
Confirmed resolved for me.

**Comment by nicolaor** [comment]
@davidmrdavid works for me too, thanks!

**Comment by th3ragex** [comment]
@davidmrdavid locking works now. Thank you!

**Comment by bachuv** [comment]
Closing this issue as it is resolved in hotfix release [durabletask-dotnet v1.2.4](https://github.com/microsoft/durabletask-dotnet/releases/tag/v1.2.4).
